test/fixtures/transaction_builder.json

```diff
@@ -1547,6 +1547,27 @@
         ]
       }
     ],
+    "fromTransactionSequential": [
+      {
+        "description": "Transaction w/ P2SH(P2MS 2/3) -> ?",
+        "network": "testnet",
+        "txHex": "0100000001b033b2214568b49fda417371aba0634b0303a2b6a19884c25d03d0b91bdbe231000000006f000000004c6952210258db1bb3801f1ecde47602143beaeb9cac93251724b8e589fae5c08c1a399a9121038e803e3d84cfc821cc8bf46233a9c2bb359d529db0bcdd3f1a4f38678dd02d7f2103b83e59d848407d7f62a82c99905f5ca3e8e8f5d6400eb78a0b4b067aea0720d953aeffffffff0200e1f5050000000017a914a9974100aeee974a20cda9a2f545704a0ab54fdc87c72831010000000017a9149f57a6712ef023f85ffac631ed4263b977b2d0678700000000",
```

Comment:
@omrip30 I assume it is OK for me to add this (your transaction) as a test fixture?

Reply:
No problem, I created this especially for this example :)